1.9/api/versioning.md

```diff
@@ -0,0 +1,31 @@
+---
+post_title: Versioning
+menu_order: 2
+---
+
+The DC/OS API is backed by many loosely coupled components, some of which are standalone projects and some of which were designed exclusively for DC/OS. Because of this, several types of versioning mechanisms are supported: component, route, and resource versioning.
```

Comment:
`...some are standalone projects and other are designed exclusively for DC/OS. Because of this, a variety of versioning mechanisms are supported: component, route, and resource versioning.`